### PR TITLE
Promoting to the latest version of guava due to CVE-2018-10237

### PR DESCRIPTION
**Pull request by petrdousa**
I would like to ask you to promote version of guava to at least version 25.0-jre in order to eliminate the eager allocation of the arrays. 

More details you may find here: https://github.com/google/guava/wiki/CVE-2018-10237#cve-2018-10237

### CODE REVIEW

**Comment by petrdousa** [comment]
Related issue: [215#issue-360280639](https://github.com/spotify/dockerfile-maven/issues/215#issue-360280639)

**Comment by petrdousa** [comment]
No complain it is relevant to this plugin but it is reasonable to use bug-free dependencies.

Thanks for considering it.

File: plugin/pom.xml [hunk 1]
```diff
@@ -43,7 +43,7 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <version>20.0</version>
+      <version>25.0-android</version>
```
Comment (davidxia) [inline plugin/pom.xml]: How about using [26.0-jre](https://search.maven.org/artifact/com.google.guava/guava/26.0-jre/bundle) and only supporting java 8+?

**Comment by davidxia** [comment]
No longer needed after [this PR](https://github.com/spotify/dockerfile-maven/pull/230/files#diff-d595a8236f587264b34d376cb6831780R56).

**Pull request closed without merging**